src/Sulu/Bundle/EventLogBundle/Collector/DocumentDomainEventCollector.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\EventLogBundle\Collector;
```

Comment:
Should we use the `UserInterface`/`Domain`/`Application`/`Infrastructure` structure in this bundle or should we keep consistent with the other bundles? @sulu/core-team